src/applications/caregivers/components/FormFields/AddressWithAutofill.jsx

```diff
@@ -120,16 +113,14 @@ const PrimaryAddressWithAutofill = props => {
       inputLabel={inputLabelMap[props.name]}
     />
   ) : (
-    <fieldset className="cg-address-with-autofill vads-u-margin-y--2">
-      <legend className="vads-u-font-family--sans vads-u-font-weight--normal vads-u-font-size--base vads-u-line-height--4 vads-u-display--block">
-        <p>{content['caregiver-address-description--vet-home']}</p>
-        <p className="va-address-block vads-u-margin-left--0">
-          {veteranAddress.street} {veteranAddress.street2}
-          <br />
-          {veteranAddress.city}, {veteranAddress.state}{' '}
-          {veteranAddress.postalCode}
-        </p>
-      </legend>
+    <div className="cg-address-with-autofill">
```

Comment:
I noticed during debugging that we had nested fieldsets when this was implemented, which creates an a11y issue. Adjusting the markup to remove that fieldset nesting.